src/styles/ftva/_card-meta.scss

```diff
@@ -29,6 +31,25 @@
     margin: 0 0 24px;
   }
 
+  .custom-title {
+    :deep(.rich-text) {
+      max-width: none;
+      margin: -20px 0px;
```

Comment:
@jendiamond Rather than use negative values on RichText's margins, remove/adjust the margins on RichText's inner elements.